### ISSUE TITLE
Custom options for Akvo Slider

### ISSUE THREAD

**Issue by designbychris** [akvo/Akvo-Theme#40]
Hi Sam :) 

Can you fix it so that on pages the admins have an option to make the header shorter. So one size is the standard height that it currently is. But if they tick a tickbox in the backend on the page they want. It reduces the height of the title area to being like the one here: https://designbychris.co.uk/akvo/newsite/single3.php ?

Essentially making the height of the title area 450px if the tick box is ticked.

Does that make sense?

**Comment by designbychris** [comment]
Another thing we need to add to the header area is a subtitle like on this page: https://designbychris.co.uk/akvo/newsite/europe.php If we can have an optional subtitle to display below the main one like this. That would be perfect :)

**Comment by samvthom16** [comment]
@designbychris this is my suggestion: to do the above customization the header through the page widget we have already created. Page settings should be left as it is to show default behaviour. For any custom changes, one can add the header image through the hero widget and corresponding custom styles from our theme.

**Comment by designbychris** [comment]
I honestly thought that it would be easier to just add an override on the area where you have show/hide the header, to change it to small/large.

Same with the subtitle.. I would use if statements for if the field is filled (same as the one for overriding the header text you have currently) to have it display the subtitle if its populated.

Is this not possible?

**Comment by designbychris** [comment]
Fair enough.. I think that the way I suggested allows for more control, and is pretty simple, as its just a text field, not a builder element. It will also require less coding to get it working I would have thought than getting the builder element to have the size built in for the smaller headings.. as we can't allow the admin to change this.. It needs to be set and kept the same height. I'm happy to open this discussion out to Linda to see which way she would prefer.

**Comment by designbychris** [comment]
we could always just have a class set to force the height in the row. That might work?

**Comment by samvthom16** [comment]
I'm already using a class .item-narrow to make the header shorter. The class gets added automatically through the widget options. Do have a look at the widget settings for reference https://staging3.akvo.org/wp-admin/post.php?post=42032&action=edit&lang=en

**Comment by designbychris** [comment]
Perfect :D winner winner chicken dinner :D
